app/assets/v2/js/vue-components.js

```diff
@@ -216,20 +236,19 @@ Vue.component('project-directory', {
         delete vm.params['search'];
       }
 
-      let searchParams = new URLSearchParams(vm.params);
+      const searchParams = new URLSearchParams(vm.params);
 
-      let apiUrlProjects = `/api/v0.1/projects_fetch/?${searchParams.toString()}`;
+      const apiUrlProjects = `/api/v0.1/projects_fetch/?${searchParams.toString()}`;
```

Comment:
In this PR you are removing this endpoint on urls.py (maybe is already in another area of the pr but not showing here)

Reply:
ohh I see it is now on HackathonProjectsViewSet right?